### ISSUE TITLE
Handle changed address from Paypal

### ISSUE THREAD

**Issue by digitalWestie** [solidusio/solidus_paypal_braintree#163]
Related to issue #106 "Handle changed address from Apple Pay sheet" - it's possible for the user to select PayPal, bounce off to PayPal and change their address, potentially to an unsupported shipping location.

**Comment by mntmn** [comment]
Yep, similar problem here. A company name that is entered in Solidus (when the "company" config of Solidus is enabled) gets lost in the process. Also, address validation can silently fail after returning from Paypal. I'd prefer an option to disable the address import from Paypal/Braintree completely.

**Comment by mntmn** [comment]
The following decorator works for me to ignore address changes coming from Braintree transactions:

```
module MntmnTransactionImportDecorator
  def address
    nil
  end
end

SolidusPaypalBraintree::TransactionImport.prepend(MntmnTransactionImportDecorator)
```

**Comment by digitalWestie** [comment]
Worked perfectly @mntmn !
